src/megaapi_impl.cpp

```diff
@@ -22380,27 +22380,45 @@ const char *MegaPricingPrivate::getAndroidID(int productIndex)
     return NULL;
 }
 
+bool MegaPricingPrivate::isBusinessType(int productIndex)
+{
+    if((unsigned)productIndex < type.size())
+        return type[productIndex];
+
+    return false;
+}
+
+int MegaPricingPrivate::getAmountMonth(int productIndex)
+{
+    if((unsigned)productIndex < amountMonth.size())
```

Comment:
Castings with `static_cast<size_t>`, please.
```suggestion
    if (static_cast<size_t>(productIndex) < amountMonth.size())
```